integrations/deploy.yaml

```diff
@@ -562,14 +558,50 @@
   keywords:
     - windows
   install_description: |
-    1. Install [Windows Exporter](https://github.com/prometheus-community/windows_exporter) on every Windows host you want to monitor.
-    2. Install Netdata agent on Linux, FreeBSD or Mac.
-    3. Configure Netdata to collect data remotely from your Windows hosts by adding one job per host to windows.conf file. See the [configuration section](https://learn.netdata.cloud/docs/data-collection/monitor-anything/System%20Metrics/Windows-machines#configuration) for details.
-    4. Enable [virtual nodes](https://learn.netdata.cloud/docs/data-collection/windows-systems#virtual-nodes) configuration so the windows nodes are displayed as separate nodes.
+    Netdata offers a convenient Windows installer for easy setup. This executable provides two distinct installation modes, outlined below.
```

Comment:
This should also have beta/paid-only notes.

And the silent mode

Reply:
cc @netdata/product